posts/regular-expression-patterns.md

```diff
@@ -5,6 +5,7 @@ tags: [tutorial, search, Sourcegraph]
 description: Learn how to use regular expression search patterns to search code on Sourcegraph.
 image: /headers/sourcegraph-learn-header-5.svg
 socialImage: /headers/sourcegraph-learn-header-5.png
+alternateTitle: Regular Expressions, How To Search with Sourcegraph using Regular Expression Patterns, Sourcegraph
```

Comment:
To optimize alternative/browser titles for SEO, we would use different keywords, as, for example "Using RegEx with Sourcegraph" that way "regex" in the browser title and "regular expressions" in the normal title would support SEO